.github/workflows/jenkins-security-scan.yml

```diff
@@ -17,4 +17,4 @@ jobs:
     uses: jenkins-infra/jenkins-security-scan/.github/workflows/jenkins-security-scan.yaml@v2
     with:
       java-cache: maven # Optionally enable use of a build dependency cache. Specify 'maven' or 'gradle' as appropriate.
-      java-version: 11 # What version of Java to set up for the build.
+      java-version: 17 # What version of Java to set up for the build.
```

Comment:
Probably best to go with

```suggestion
```

because it's optional since today :)

Reply:
Yeah, should have thought of that myself 👍🏼 
Good thing that GitHub API rate limit stopped me from creating more PR today. Next ones will rely on the default if possible.

Reply:
> thought of that myself

No worries, I only did that ~3 hours ago and you had no way to know that I'd finally get around to do this :)